prompt/git-k8s.nu

```diff
@@ -0,0 +1,438 @@
+def bright_cyan [] {
+  each { |it| $"(ansi -e '96m')($it)(ansi reset)" }
```

Comment:
if you look the names in `ansi --list`, you'll see that it's not necessary to do things like `ansi -e '96m'` because there are already pre-defined names. you could use `ansi light_cyan` or `ansi lc` for short.

Reply:
This section is copied over, I haven't looked at it closely yet.